src/ai/backend/manager/models/alembic/versions/51dddd79aa21_add_logs_column_on_kernel_table.py

```diff
@@ -0,0 +1,24 @@
+"""Add logs column on kernel table
+
+Revision ID: 51dddd79aa21
+Revises: 1e8531583e20
+Create Date: 2020-02-11 14:45:55.496745
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = '51dddd79aa21'
+down_revision = '1e8531583e20'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    op.add_column('kernels', sa.Column('container_log', sa.String, nullable=True))
```

Comment:
Let's change this column type to [`sa.LargeBinary`](https://docs.sqlalchemy.org/en/13/core/type_basics.html#sqlalchemy.types.LargeBinary) to store the given log data as-is.
Since the log may contain non-Unicode characters and garbled outputs (due to mishandling of encodings in the user codes, etc.), it would be safer to use binary types to store them.
This will also allow appropriate conversion and streaming in the database backends if available.